src/main/utils/LaunchDarkly.ts

```diff
@@ -0,0 +1,47 @@
+import config from 'config';
+import launchDarkly, { LDClient, LDOptions, LDUser } from 'launchdarkly-node-server-sdk';
+import {FeatureFlagClient} from '../types/FeatureFlagClient';
+
+class LaunchDarkly implements FeatureFlagClient {
+  private static instance: LaunchDarkly
+  private readonly client: LDClient;
+  private readonly ldUser: LDUser;
+
+  private constructor() {
+    const sdkKey: string = config.get('launchDarkly.sdkKey');
+    const options: LDOptions = config.get('featureToggles.enabled') ? { diagnosticOptOut: true } : { offline: true };
+    this.ldUser = { key: 'fact-admin' };
+    this.client = launchDarkly.init(sdkKey, options);
+  }
+
+  public static getInstance(): LaunchDarkly {
+    if (!LaunchDarkly.instance) {
+      LaunchDarkly.instance = new LaunchDarkly();
+    }
+
+    return LaunchDarkly.instance;
+  }
+
+  public async getFlagValue(flag: string, defaultValue: boolean): Promise<boolean> {
+    return this.client.variation(flag, this.ldUser, defaultValue);
```

Comment:
I'm trying to work out when the defaultValue would be used. Is this when the flag/key doesn't exist?

Reply:
Yep, defaultValue is the fallback when the flag doesn't exist, or there's issues with contacting the 3rd party for the flag's value.